scripts/mellanox_buffer_migrator.py

```diff
@@ -243,56 +337,143 @@ def mlnx_default_buffer_parameters(self, db_version, table):
 
         return self.mellanox_default_parameter[db_version].get(table)
 
+    def mlnx_migrate_map_old_pool_to_new(self, pool_mapping, pool_convert_map, old_config_name):
+        new_config_name = None
+        if pool_mapping:
+            new_config_map = pool_mapping.get(old_config_name)
+            if type(new_config_map) is tuple:
+                method, mapname = new_config_map
+                if method == "sku":
+                    skumap = pool_convert_map.get(mapname)
```

Comment:
Not sure whether we need this because in any case, it shouldn't be None. If it were None, there must be an error in the param definition.
In this sense, I would like to have it thrown an exception so that the developer is able to investigate it by checking the call stack.